daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -339,6 +355,7 @@ func ValidateLogOpt(cfg map[string]string) error {
 		switch key {
 		case logGroupKey:
 		case logStreamKey:
+		case logStreamTemplateKey:
```

Comment:
Can you validate that both `logStreamKey` and `logStreamTemplateKey` are not defined?  They should be mutually-exclusive options since only one would ever actually be used.

If you don't want to replace the `:` in the value for `awslogs-stream-template`, can you reject the option here?

A unit test would be good to add for this logic.